docs/reference/cli.md

```
@@ -0,0 +1,271 @@
+giant check-data gtex
+
+# Verbose output showing missing files
+giant check-data tcga -v
+
+# JSON output
+giant check-data panda --json
+```
+
+### Output
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)